sdk/translation/Azure.AI.Translation.Document/samples/README.md

```diff
@@ -5,7 +5,7 @@ languages:
 products:
 - azure
 - azure-cognitive-services
-- azure-document-translation
+- azure-translator-text
```

Comment:
I think we should use `azure-translator` as it is more generic. `azure-translator-text` might only cover real-time translation, which this SDK is not targeting.
I am following with their PM to validate `azure-translator` is the right one

Reply:
FYI @kristapratico

Reply:
Agree, was going to update to azure-translator. Let us know what you learn, @maririos !

Reply:
ok. The service team confirmed that we should use `azure-translator`

Reply:
Done